src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -1,14 +1,13 @@
 package uk.gov.companieshouse.api.strikeoffobjections.service.impl;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
```

Comment:
Another unused import

Reply:
I have no idea howe these keep showing up I am deleting them all and then there are more of them.  I have just cleaned up EmailsService - the ide was showng none.

Reply:
Everyone should have the SonarLint plugin installed (in intelliJ), there should be an equivalent for Eclipse

Reply:
Yep. That plug-in works in Eclipse too. I have it installed. If you are using Eclipse @mattch1 , SHIFT-OPTION-O cleans up your imports.